pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -655,6 +652,27 @@ public void run() {
                         }
                     }
                 });
+                Timer statusCheckTimer = new Timer();
+                statusCheckTimer.scheduleAtFixedRate(new TimerTask() {
```

Comment:
Since this is never stopped, can this keep the process hanging on shutdown?

Reply:
Good catch. Fixed